js/components/adPanel.js

```diff
@@ -133,6 +133,7 @@ var AdPanel = React.createClass({
     return (
       <div className="oo-ad-screen-panel">
         {spinner}
+        <div className="oo-ad-screen-panel-click-layer"></div>
```

Comment:
I used a separate element here because, if I changed the z-index of `oo-ad-screen-panel`, I would invalidate the z-index of `oo-ad-top-bar`. With this approach I was able to fix the issue without changing the z-index of `oo-ad-top-bar`.